### PR TITLE
update acceptance suites

### PR DESCRIPTION
**Pull request by patrick-wright**
@sersut 

1) fixes `kitchen` exec => `bundle exec kitchen`
2) allows `KITCHEN_INSTANCES=ubuntu bundle exec chef-acceptance test` to just run ubuntu suites

I think we should  consider removing Gemfile.lock
Note Gemfile is referencing a local chef-acceptance path

### CODE REVIEW

File: acceptance/Gemfile [hunk 1]
```diff
@@ -2,7 +2,8 @@ source "https://rubygems.org"
 
 gem "mixlib-install", path: "../"
 gem "test-kitchen", github: "sersut/test-kitchen", branch: "sersut/mixlib-install-update"
-gem "chef-acceptance", github: "chef/chef-acceptance"
+# gem "chef-acceptance", github: "chef/chef-acceptance"
+gem "chef-acceptance", path: "../../chef-acceptance"
```
Comment (sersut) [inline acceptance/Gemfile]: We should revert this.

File: acceptance/current/.kitchen.yml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 driver:
   name: vagrant
-  forward_agent: yes
```
Comment (sersut) [inline acceptance/current/.kitchen.yml]: Any reason we're removing `forward_agent` ?

**Comment by sersut** [comment]
Yeah let's remove Gemfile.lock :smile:

File: acceptance/current/.kitchen.yml (hunk 2 above)
Comment (patrick-wright) [inline acceptance/current/.kitchen.yml]: I thought that was just needed for vbox issues for some reason.  We can put it back.

**Pull request merged**